### ISSUE TITLE
a11y support

### ISSUE THREAD

**Issue by Link2Twenty** [Zecat/paper-stepper#19]
Just thinking it might be worth making it so your paper-horizontal-step's and paper-vertical-step's can be tabbed to and selected with space.

There is a good video on how to do it easily https://www.youtube.com/watch?v=REVxMvdBYMw

**Comment by Zecat** [comment]
You're right, I think a good start for accessibility should be to implement iron-menu-behavior for the vertical and iron-menubar-behavior for the horizontal stepper.
